lib/loopback.js

```diff
@@ -76,6 +95,10 @@ function createApplication() {
   app.connector('memory', loopback.Memory);
   app.connector('remote', loopback.Remote);
 
+  // setup the app registry
+  var registry = app.registry = new Registry();
+  require('./builtin-models')(registry);
+
```

Comment:
> However, I still think it is a good idea to let the application decide whether it wants local or global model registry.

I don't think we need to support `app.set()` and `config.localRegsitry`... Thats overkill IMO. Also its impractical. See more on comments below RE: specific implementation for the flag.